driven/courses/uiuc_courses.go

```diff
@@ -0,0 +1,298 @@
+/*
+ *   Copyright (c) 2020 Board of Trustees of the University of Illinois.
+ *   All rights reserved.
+
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+
+ *   http://www.apache.org/licenses/LICENSE-2.0
+
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package courses
+
+import (
+	model "apigateway/core/model"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type campusData struct {
```

Comment:
The way you have it here is fine, but you could also move these structs to a `models.go` file in the same `courses` package to improve the readability of this file.